aks-hci/known-issues/0A930D94-52C8-443B-9718-BE2EE6503355.yml

```diff
@@ -4,15 +4,15 @@
 "Reviewer": "abha"
 "ms.topic": "faq"
 "Schema": "doc"
-"Metatitle": "Known issues in Windows Admin Center for Azure Kubernetes Service on Azure Stack HCI and Windows Server"
-"Metadescription": "Known issues in Windows Admin Center for Azure Kubernetes Service on Azure Stack HCI and Windows Server"
+"Metatitle": "Known issues in Windows Admin Center for AKS hybrid"
+"Metadescription": "Known issues in Windows Admin Center for AKS hybrid"
 "title": "Fix issues in Windows Admin Center"
 "Summary": |
-  This article describes known issues with Windows Admin Center in Azure Kubernetes Service on Azure Stack HCI and Windows Server. You can also review [upgrade](/azure-stack/aks-hci/known-issues) and [installation](/azure-stack/aks-hci/known-issues-installation) issues and errors.
+  This article describes known issues with Windows Admin Center in AKS hybrid. Azure Kubernetes Service hybrid deployment options ("AKS hybrid") offer a fully supported container platform that can run cloud-native applications on the [Kubernetes container orchestration platform](https://kubernetes.io/). The architecture supports running virtualized Windows and Linux workloads. You can also review [upgrade](/azure-stack/aks-hci/known-issues) and [installation](/azure-stack/aks-hci/known-issues-installation) issues and errors.
 "Additionalcontent": |
   ## Next steps
   - [Troubleshooting overview](troubleshoot-overview.md)
   - [Installation issues and errors](/azure-stack/aks-hci/known-issues-installation)
   - [Upgrade known issues](/azure-stack/aks-hci/known-issues-upgrade)
 
-  If you continue to run into problems when you're using AKS on Azure Stack HCI and Windows Server, you can file bugs through [GitHub](https://aka.ms/aks-hci-issues).
+  If you continue to run into problems when you're using AKS on Azure Stack HCI or Windows Server, you can file bugs through [GitHub](https://aka.ms/aks-hci-issues).
```

Comment:
This isn't right -- it's not "AKS on Azure Stack HCI **OR** Windows Server". The product name used to be "AKS on Azure Stack HCI and Windows Server". So please don't change it to "AKS on Azure Stack HCI or Windows Server," that is incorrect.

Reply:
Fixed.